modules/fasta_writer.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+process FASTA_WRITER {
+    publishDir "$launchDir/$params.outdir/"
+    stageOutMode = 'copy'
```

Comment:
Why do you copy the file out?

Reply:
I have replaced the stageOutMode by publishDir mode in all relevant nextflow module files